integrations/amazon_bedrock/src/haystack_integrations/components/generators/amazon_bedrock/generator.py

```diff
@@ -33,16 +33,18 @@
 @component
 class AmazonBedrockGenerator:
     """
-    Generator based on a Hugging Face model.
-    This component provides an interface to generate text using a Hugging Face model that runs locally.
+    `AmazonBedrockGenerator` enables text generation via Amazon Bedrock hosted LLMs.
+
+    For example, to use the Anthropic Claude model, simply initialize the `AmazonBedrockGenerator` with the
+    'anthropic.claude-v2' model name. Provide AWS credentials either via local AWS profile or directly via
+    `aws_access_key_id`, `aws_secret_access_key`, `aws_session_token`, and `aws_region_name` parameters.
 
     Usage example:
     ```python
-    from amazon_bedrock_haystack.generators.amazon_bedrock import AmazonBedrockGenerator
+    from haystack_integrations.components.generators.amazon_bedrock import AmazonBedrockGenerator
 
     generator = AmazonBedrockGenerator(
```

Comment:
The AWS creds are no longer strings, they are either `Secret` or env var - you can omit them.

```suggestion
generator = AmazonBedrockGenerator(
        model="anthropic.claude-v2",
        max_length=99,
)
```